### PR TITLE
PLAT-685: add checks of VCallResult

### PR DESCRIPTION
**Pull request by ivanshibitov**
Add logic checks of VCallResult

### CODE REVIEW

File: ledger-core/insolar/payload/validation.go [hunk 1]
```diff
@@ -22,6 +22,14 @@ func isSpecialTimePulseBefore(pn pulse.Number, before pulse.Number) bool {
 	return pn.IsSpecial() || pn.IsTimePulse() && pn.IsBefore(before)
 }
 
+func isTimePulseBeforeOrEq(pn pulse.Number, before pulse.Number) bool {
+	return pn.IsTimePulse() && pn.IsBeforeOrEq(before)
+}
+
+func isSpecialTimePulseBeforeOrEq(pn pulse.Number, before pulse.Number) bool {
```
Comment (cyraxred) [inline ledger-core/insolar/payload/validation.go:29]: should be
isSpecialOrTimePulseBeforeOrEq

Comment (bigbes) [reply ledger-core/insolar/payload/validation.go:29]: I'll rename it in my PR

File: ledger-core/virtual/integration/deduplication/constructor_test.go [hunk 2]
```diff
@@ -259,6 +259,9 @@ func (test *DeduplicationDifferentPulsesCase) run(t *testing.T) {
 	if test.ExecuteShouldHaveError != "" {
 		foundError = test.Server.Journal.Wait(func(event debuglogger.UpdateEvent) bool {
 			if event.Data.Error != nil {
+				if _, ok := event.Data.Error.(throw.StackTraceHolder); !ok {
```
Comment (cyraxred) [inline ledger-core/virtual/integration/deduplication/constructor_test.go:262]: why do you need this check?
DeepestStackTraceOf will then just return nil

Comment (ivanshibitov) [reply ledger-core/virtual/integration/deduplication/constructor_test.go:262]: yes, fix it

**Pull request merged**